tests/test_widgets.py

```diff
@@ -185,6 +185,34 @@ class W(SuffixedMultiWidget):
         }, {}, 'price')
         self.assertEqual(result, ['1', 'lt'])
 
+    def test_value_omitted_from_data(self):
+        class A(SuffixedMultiWidget):
+            suffixes = ['_b']
```

Comment:
The `_` is unexpected to me here. Just wondering if others would be surprised as well.